python/cuml/test/conftest.py

```diff
@@ -26,9 +26,12 @@ def pytest_configure(config):
 
 @pytest.fixture(scope="module")
 def nlp_20news():
-    twenty_train = fetch_20newsgroups(subset='train',
-                                      shuffle=True,
-                                      random_state=42)
+    try:
```

Comment:
I think you can simplify this even further by decorating the function with: ```
@pytest.mark.xfail(raises=IOError)
```

then the body does not have to change

Reply:
It's the first thing that I tried but it didn't work because this function is a pytest fixture and not a test in itself so I think it can't be expected to fail.

I would have to decorate all the tests of `test_naive_bayes.py` for this to work.

Reply:
Good point. I think it makes sense as is then. Thanks!